code/game/objects/structures/crates_lockers/closets/secure/personal.dm

```diff
@@ -43,13 +43,26 @@
 		to_chat(user, "<span class='warning'>Invalid identification card.</span>")
 		return
 
+	if(opened)
+		to_chat(user, "<span class='notice'>Close the locker first.</span>")
+		return
+
```

Comment:
```suggestion
```
It will return at the beginning of this proc if it is opened

Reply:
Removed unnecessary if statement.